src/astro/files/locations/gcs.py

```diff
@@ -0,0 +1,34 @@
+from typing import Dict, List
+from urllib.parse import urlparse, urlunparse
+
+from astro.constants import FileLocation
+from astro.files.locations.base import Location
+from astro.utils.dependencies import GCSClient, GCSHook, gcs
+
+
+class GS(Location):
```

Comment:
`GCS` please as it is called Google Cloud Storage

```suggestion
class GCS(Location):
```

Reply:
Done